### PR TITLE
SAK-49506 Samigo autosubmit events are not localized in the event log 

### PR DESCRIPTION
**Pull request by mpellicer**

### CODE REVIEW

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/LinearAccessDeliveryActionListener.java [hunk 1]
```diff
@@ -194,7 +194,7 @@ public void processAction(ActionEvent ae) throws
           }
           eventLogData.setSiteId(site_id);
           
-          eventLogData.setErrorMsg(eventLogMessages.getString("no_submission"));
+          eventLogData.setErrorMsg("no_submission_user_no_submit");
```
Comment (bgarciaentornos) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/LinearAccessDeliveryActionListener.java:197]: I think you could remove the eventLogMessages and the ResourceBundle import in this file like you do in the other ones.

File: samigo/samigo-app/src/webapp/jsf/event/eventLog.jsp [hunk 2]
```diff
@@ -140,8 +140,12 @@
 	  </f:facet>
 
 	 <h:panelGroup styleClass="#{eventLog.isDeleted(log.assessmentId) ? 'eventLogDeleted' : ''}">
-	  <h:outputText value="#{log.errorMsg}" rendered="#{!log.isNoErrors}"/>
-	    <h:outputText value="#{log.errorMsg}" styleClass="prePopulateText" rendered="#{log.isNoErrors}"/>
+	   <h:outputText value="#{log.errorMsg}" rendered="#{!log.isNoErrors}">
+         <f:converter converterId="org.sakaiproject.tool.assessment.jsf.convert.EventLogConverter" />
```
Comment (bgarciaentornos) [inline samigo/samigo-app/src/webapp/jsf/event/eventLog.jsp:144]: Fix spacing

File: samigo/samigo-app/src/webapp/jsf/event/eventLog.jsp [hunk 3]
```diff
@@ -140,8 +140,12 @@
...
+	   </h:outputText>
+	   <h:outputText value="#{log.errorMsg}" styleClass="prePopulateText" rendered="#{log.isNoErrors}">
+         <f:converter converterId="org.sakaiproject.tool.assessment.jsf.convert.EventLogConverter" />
```
Comment (bgarciaentornos) [inline samigo/samigo-app/src/webapp/jsf/event/eventLog.jsp:147]: Fix spacing

**Review by bgarciaentornos — CHANGES_REQUESTED** [review]

File: samigo/samigo-app/src/webapp/jsf/event/eventLog.jsp (hunk 2 above)
Comment (mpellicer) [reply samigo/samigo-app/src/webapp/jsf/event/eventLog.jsp:144]: Okay

File: samigo/samigo-app/src/webapp/jsf/event/eventLog.jsp (hunk 3 above)
Comment (mpellicer) [reply samigo/samigo-app/src/webapp/jsf/event/eventLog.jsp:147]: Okay

**Review by bgarciaentornos — APPROVED** [review]
Thanks for the fixes!!

**Pull request merged**